1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -20,11 +20,23 @@ instead!
    Use `console.log` to display the result.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
-  // TODO complete this object
+  beers: 15.75,
+  chips: 40.54,
+  rice: 50.32,
+  water: 30.76,
+  bread: 40.98,
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(myObject) {
+  let sum = 0;
+
+  for (const el in myObject) {
+    if (Object.prototype.hasOwnProperty.call(myObject, el)) {
```

Comment:
You can use `Object.hasOwnProperty.call` - without "prototype" here instead.

Reply:
I removed it and used like "Object.hasOwn.call"